tests/cluster_tests/compute_test.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# Copyright 2023 Battelle Energy Alliance, LLC
+#
```

Comment:
It seems to me this is inconsistent with RAVEN, I mean change 2023 to 2017 here?

Reply:
Hm, changed, but I am not sure it is needed.